test/BotTester.spec.ts

```diff
@@ -302,6 +302,31 @@ describe('BotTester', () => {
             .runTest();
     });
 //```
+
+//# Can wait between test steps
+//```javascript
+    it('can wait between test steps', () => {
+        const delay = 1000;
+        let beforeDelayTime;
+        let afterDelayTime;
+
+        bot.dialog('/', (session) => {
+            // send only numbers for this test case ....
+            if (afterDelayTime - beforeDelayTime >= delay) {
+              session.send('i waited some time');
+            }
+
+        });
+
+        return new BotTester(bot)
+            .then(() => beforeDelayTime = Date.now())
+            .wait(delay)
+            .then(() => afterDelayTime = Date.now())
+            .sendMessageToBot('have you waited ?', 'i waited some time')
+            .runTest();
```

Comment:
you need to actually ensure that there is a ~1000 ms delay difference between the two times for this test to pass correctly.

Reply:
I actually completely missed that you had the check integrated. I would update it though to explicitly check the before and after times instead of encoding it in the bot response

Reply:
Don't forget to update the README.md accordingly. You added in //``` for it as needed, I just don't have an automated doc builder for that portion yet, so its a manual process